org.unitystation.StationHub.metainfo.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ideally should be part of https://github.com/unitystation/stationhub – it's not Flathub specific.